dev/com.ibm.ws.cdi.internal/resources/com/ibm/ws/cdi/internal/resources/CDI.nlsprops

```diff
@@ -88,3 +88,16 @@ bce.not.loadable.CWOWB1013E.useraction=Ensure the class is present in the applic
 bce.does.not.implement.bce.CWOWB1014E=CWOWB1014E: The {0} class does not implement the BuildCompatibleExtension interface.
 bce.does.not.implement.bce.CWOWB1014E.explanation=The application includes a META-INF/services/jakarta.enterprise.inject.build.compatible.spi.BuildCompatibleExtension file that includes the class, but the class does not implement jakarta.enterprise.inject.build.compatible.spi.BuildCompatibleExtension
 bce.does.not.implement.bce.CWOWB1014E.useraction=Change the class so that it implements jakarta.enterprise.inject.build.compatible.spi.BuildCompatibleExtension.
+
+cdi12.element.type.superceded.CWOWB1015W=CWOWB1015W: The configuration element type cdi12 has been superceded by the element type cdi.
+cdi12.element.type.superceded.CWOWB1015W.explanation=The cdi configuration element type completely replaces the cdi12 element. The cdi configuration element type contains all the same options as the cdi12 one, plus some new options.
+cdi12.element.type.superceded.CWOWB1015W.useraction=Change your server.xml configuration to use the cdi element type instead of the cdi12 element type.
+
+cdiEmptyBeansXMLExplicitArchive.ignored.CWOWB1016W=CWOWB1016W: The attribute emptyBeansXMLExplicitArchive of the element type cdi is only supported on CDI 4.0 or newer. This attribute will be ignored.
```

Comment:
suggested edit.

The emptyBeansXMLExplicitArchive  attribute of the cdi configuration element is supported only on CDI 4.0 or later. This attribute is ignored.